docs/tutorials/build_basic_model.ipynb

```diff
@@ -9,18 +9,39 @@
     "This _Build a basic model_ tutorial illustrates how to build a simple D-Flow FM model from scratch using [HYDROLIB-core](https://github.com/Deltares/HYDROLIB-core). "
    ]
```

Comment:
@arthurvd, I think the notebook suffices for testing whether hydrolib-core installation has succeeded. However, the imports that we use here are from an unreleased version. I assume that users will install the latest released version (0.3.1) and will not use the main. The main has the new `dflowm` package, these imports will not work with 0.3.1.

For my demo I have created a branch from release 0.3.1. You might consider doing that as well.

Reply:
You are right. I did that, and created a draft PR for the 0.3.1-compatible version.
I'll close this PR now without merging.